src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Shape {
```

Comment:
let's use abstract class to store common state, do we have common fields among our child classes?